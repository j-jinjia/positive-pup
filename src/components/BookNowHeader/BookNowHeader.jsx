import "./BookNowHeader.scss";
import Header from "../Header/Header";

const BookNowHeader = ({ headingText, subheadingText }) => {
  return (
    <div className="book-now-header">
      <Header headingText={headingText} subheadingText={subheadingText} />
    </div>
  );
};

export default BookNowHeader;
