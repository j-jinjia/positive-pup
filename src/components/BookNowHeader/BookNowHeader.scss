@use "../../styles/variables" as *;

.book-now-header {
  text-align: center;
  padding-left: 2rem;
  padding-right: 2rem;
  &__heading {
    color: $colour-primary;
    font-size: $font-size-h1;
    font-family: $font-header;
    margin-bottom: 0.5rem;
    text-transform: uppercase;
  }
  &__subheading {
    font-size: $font-size-h2;
    font-family: $font-header;
    font-weight: normal;
    margin-bottom: 1.75rem;
  }
}
