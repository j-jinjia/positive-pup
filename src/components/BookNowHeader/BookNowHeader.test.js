import { customRender } from "../../utils/testUtils";
import BookNowHeader from "./BookNowHeader";

it("should render the BookNowHeader component", () => {
  const { container } = customRender(<BookNowHeader />);

  expect(container).toMatchSnapshot();
});
