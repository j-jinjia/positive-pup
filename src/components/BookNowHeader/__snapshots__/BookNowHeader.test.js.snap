// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the BookNowHeader component 1`] = `
<div>
  <div
    class="book-now-header"
  >
    <div
      class="headerheader"
    >
      <h1
        class="header__heading"
      >
         
         
      </h1>
      <h2
        class="header__subheading"
      />
    </div>
  </div>
</div>
`;
